src/copick/models.py

```diff
@@ -1424,6 +1489,59 @@ def refresh(self) -> None:
         """Refresh the points from storage."""
         self.meta = self.load()
 
+    def numpy(self) -> Tuple[np.ndarray, np.ndarray]:
+        """Return the points as a [N, 3] numpy array (N, [x, y, z]) and the transforms as a [N, 4, 4] numpy array.
+        Format of the transforms is:
+                ```
+                [[rxx, rxy, rxz, tx],
+                 [ryx, ryy, ryz, ty],
+                 [rzx, rzy, rzz, tz],
+                 [  0,   0,   0,  1]]
+                ```
+
+        Returns:
+            Tuple[np.ndarray, np.ndarray]: The picks and transforms as numpy arrays.
+        """
+
+        points = np.zeros((len(self.points), 3))
+        transforms = np.zeros((len(self.points), 4, 4))
+
+        for i, p in enumerate(self.points):
+            points[i, :] = np.array([p.location.x, p.location.y, p.location.z])
+            transforms[i, :, :] = p.transformation
+
+        return points, transforms
```

Comment:
very glad to see this function for picks. as we see how user experiences go, it might be worthwhile to make this the default internal representation. if we eventually want to support querying for picks in regions then we'll want to use a numpy array to leverage libraries with faster spatial queries.